src/SplitScreen/SplitScreen.js

```diff
@@ -0,0 +1,276 @@
+          case 'change:rotation':
+            return thisView.setRotation(view.getRotation())
+          default:
+            return ''
+        }
+      }
+
+      this.syncMapEvents(thisView, event.type)
+    })
+  }
+
+  removeMap = () => {
+    const { maps, onMapRemoved, visibleMapCount } = this.props
+    const mapIndex = visibleMapCount - 1
+    const mapToRemove = maps[mapIndex]
+
+    mapToRemove.setVisibleState(false)
+    mapToRemove.setSyncedState(false)
+    mapToRemove.setLayerGroup(new olLayerGroup())
+
+    // update UI & alert component parent of change
+    this.handleMapChange(visibleMapCount - 1)
+
+    if (visibleMapCount - 1 === 2) this.setState({ startPosition: window.innerWidth / 2 })
+    onMapRemoved(mapIndex)
+  }
+
+  addMap = () => {
+    const { onMapAdded, maps, visibleMapCount } = this.props
+    const nextMapIdx = visibleMapCount
+    const prevMap = maps[nextMapIdx - 1]
+    const mapToAdd = maps[nextMapIdx]
+    const prevMapLayer = prevMap.getLayerGroup().getLayers().item(0)
+
+    mapToAdd.setVisibleState(true)
+    mapToAdd.setSyncedState(true)
+
+    mapToAdd.getLayers().setAt(0, prevMapLayer)
+
+    mapToAdd.getView().setCenter(prevMap.getView().getCenter())
+    mapToAdd.getView().setZoom(prevMap.getView().getZoom())
+
+    // first de-sync everything (this prevents double binding sync events)
+    maps.map(m => this.desyncMapEvents(m.getView()))
+
+    // now get the syncable maps and then sync them
+    maps.filter(m => m.getSyncedState()).map(m => this.syncMapEvents(m.getView()))
+
+    // update UI & alert component parent of change
+    this.handleMapChange(visibleMapCount + 1)
+
+    if (visibleMapCount + 1 === 2) this.setState({ startPosition: window.innerWidth / 2 })
+    onMapAdded(nextMapIdx)
+  }
+
+  toggleSyncMap = (i) => {
+    const { maps, toggleSyncMap } = this.props
+    const currentSyncState = maps[i].getSyncedState()
+
+    // set the map sync state to the inverse
+    maps[i].setSyncedState(!currentSyncState)
+
+    // first de-sync everything (this prevents double binding sync events)
+    maps.map(m => this.desyncMapEvents(m.getView()))
+
+    // now get the syncable maps and then sync them
+    maps.filter(m => m.getSyncedState()).map(m => this.syncMapEvents(m.getView()))
+
+    // update UI & alert component parent of change
+    this.props.forceUpdate()
+    toggleSyncMap()
+  }
+
+  render () {
+    const { startPosition } = this.state
+    const { translations, maps, syncedState, visibleState, visibleMapCount } = this.props
+    const disabled = false
+
+    this.handleMapChange(visibleMapCount, false)
+
+    return (
+      <Container>
+        {disabled
+          ? <div className='alert alert-warning' role='alert'>{translations['geokit.SplitScreen.disabled']}</div>
+          : null
+        }
+        <MapDisplayContainer disabled={disabled}>
+          {visibleState.map((visible, i) => {
+            const grow = i === 0 && visibleMapCount === 3
```

Comment:
What's the meaning of `3` here? It this trying to force a new row every two or something?